WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -813,6 +813,14 @@ extension Media {
             nsError.domain == multipartEncodingFailedSampleError.domain,
             nsError.code == multipartEncodingFailedSampleError.code {
```

Comment:
@crazytonyli looking at this code, it occurred to me it would be "nicer" with a `guard` and `switch`.

Something like:

```swift
guard let nsError = error as NSError else { return false }

switch (nsError.domain, nsError.code) {
case ("Alamofire.AFError", 2): return true
case ...
default: return false
}
```

I even started down that path, with an extension on `NSError` implementing the `switch` check on `domain` and `code`. But I realized that:

1. 2 out of the 3 case currently handled will disappear with Alamofire
2. The case checking against `multipartEncodingFailedSampleError` is dead code for the same reason that resulted in #22398 

Just wanted to call this out in case it's helpful for you as the work progresses.

Reply:
I haven't made more changes to this code here, others than this PR. But this part is definitely will be changed later (after Alamofire is removed): replacing the `AFError` reference with the error domain & code originated from the post-Alamofire WordPressKit.